selendroid-server/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
     buildToolsVersion "23.0.1" // https://developer.android.com/tools/revisions/build-tools.html
 
     defaultConfig {
-        minSdkVersion 10
+        minSdkVersion 18
```

Comment:
This should remain at 10. Appium users rely on Selendroid for API 10+ automation.